man/fido_cred_set_authdata.3

```diff
@@ -157,6 +161,22 @@ The latter two are meant to be used in contexts where the
 credential's complete attestation statement is not available or
 required.
 .Pp
+The attestation object passed to
```

Comment:
it might make sense (at a later point) to add a line or two about the purpose/utility of set_attobj

Reply:
I'll add this to my TODOs.